client/src/main/java/org/apache/rocketmq/client/latency/LatencyFaultToleranceImpl.java

```diff
@@ -76,7 +76,9 @@ public String pickOneAtLeast() {
             if (half <= 0) {
                 return tmpList.get(0).getName();
             } else {
-                final int i = this.whichItemWorst.incrementAndGet() % half;
+                int i = this.whichItemWorst.incrementAndGet() % half;
+                if (i < 0)
```

Comment:
IMO,  `i` can not less than zero

Reply:
Well, I didn't make it clear in the context.

 `i` is  return from the method `ThreadLocalIndex#incrementAndGet()`, when variable `index` in this method reach to Integer.MAX_VALUE,  `++index`  will be value of `Integer.MIN_VALUE`,  its absolute value (`Math.abs(Integer.MIN_VALUE)`) is itself, that's a negative value.

Reply:
Agree with, I think it would be better to add the comment above the `if(i < 0)`,WDYT